api/app/signals/apps/users/models.py

```diff
@@ -24,12 +24,16 @@ class Profile(CreatedUpdatedModel):
     )
 
     departments = models.ManyToManyField(
-        to='signals.Department'
+        to='signals.Department',
+        related_name='profiles'
```

Comment:
When seeing this I am not sure if we want a relation than can be accessed from the department. If so than a better name would be `user_profiles` because just `profiles` reads like `department_profiles`.